src/renderer/viz/expressions/unary.js

```diff
@@ -176,6 +176,17 @@ export const Abs = genUnaryOp('abs', x => Math.abs(x), x => `abs(${x})`);
  *
  * This returns a numeric expression where 0 means `false` and 1 means `true`.
  *
+ * @example <caption>isNaN.</caption>
```

Comment:
This needs to explain what the filter is doing

Reply:
It seems that the general rule is to use just the name for all the examples in unary expressions. However, I agree with improving the example's description.